pages/partials/questionFooter.ejs

```diff
@@ -35,6 +35,29 @@
     </div>
     <% } %>
   </div>
+  <script>
+      $('form').on('submit', function(e) {
+          if (!$(this).data('submitted')) {
+              $(this).data('submitted', true);
+
+              var form = $(this);
+              form.find('.disable-on-submit').each(function (index) {
+                  // Create a disabled clone of the submit button
+                  $(this).clone(false).removeAttr('id').prop('disabled', true).insertBefore($(this));
+
+                  // Hide the actual submit button and move it to the beginning of the form
+                  $(this).hide();
+                  form.prepend($(this));
+              });
+          } else {
+              e.preventDefault();
+          }
+      });
+
+      $('a.disable-on-click').on('click', function() {
+          $(this).addClass('disabled');
+      });
+  </script>
```

Comment:
Rather than duplicating the `<script>` for both the `Freeform` and non-`Freeform` question types, could we shift it outside the entire footer `<div>` so that it's included for both question types?

Reply:
Yes! Actually I was kinda wondering why there was so much duplication already, but now I'm guessing it's to make the v2 deprecation easier... just wipe out the `else`? Anyway I will update the branch.

Reply:
Moved `<script>` outside. Squashed with old commit.